components/http_kernel/introduction.rst

```diff
@@ -339,7 +364,18 @@ of arguments that should be passed when executing that callable.
 
     b) If the argument in the controller is type-hinted with Symfony's
        :class:`Symfony\\Component\\HttpFoundation\\Request` object, then the
-       ``Request`` is passed in as the value.
+       ``Request`` is passed in as the value. If you have a custom class extending
+       the ``Request``, this is also accepted.
+
+    c) If the function or method argument is `variadic`_ and the ``Request``
+       ``attributes`` bag contains and array for that argument, they will all be
+       available through the `variadic`_ argument.
+
+    This functionality is provided by resolvers implementing the
+    :class:`Symfony\\Component\\HttpKernel\\Controller\\ArgumentValueResolverInterface`.
```

Comment:
I know :) I was asking in the wrong place if it was not the wrong namespace in the PR. It could make sense to keep this interface in the same namespace as the implementations, WDYT ?